code_generator/GdkCodeGenerator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -344,7 +452,68 @@ namespace <#= qualifiedNamespace #>
 
             public override void SendCommands(List<ComponentGroup> commandComponentGroups, Connection connection)
             {
-                // TODO: Generate command serialization
+<#
+for (var i = 0; i < commandDetailsList.Count; i++) {
+    var commandDetails = commandDetailsList[i];
+    var commandSenderType = $"{qualifiedNamespace}.{componentDetails.ComponentName}.CommandSenders.{commandDetails.CommandName}";
+    var commandResponderType = $"{qualifiedNamespace}.{componentDetails.ComponentName}.CommandResponders.{commandDetails.CommandName}";
+#>
+                {
```

Comment:
This should check if the componentGroup is empty or skip. Usefull for when we add the filtering in later